2017/firstEmailAnnouncement.txt

```diff
@@ -19,10 +19,8 @@ Sure you are! If you know the first thing about a subject, you know more than pe
 How do I register a talk?
 There's a whiteboard near the entrance; that's the schedule grid. On it, there's a list of rooms and times, and the grid starts off empty. Simply walk up to that board, find a spot, write in your topic and your name, and then show up in that room at that time to speak. It's that simple. Please be considerate when signing up for multiple talks at the start of the conference. If you see slots lingering open later on, feel free to fill them in then. This allows for more room for new talks and late adds, while also keeping you from accidentally scheduling an hour for a 25 minute talk!
 
-But perhaps you're...
-
-Shy?
-That's fine. Come on in, take a look around and get a feel for things. It's a two-day event; come by on Friday, sit in on the presentations which interest you or hang around in the lounge and network with other people. Perhaps you'll be inspired to talk about something later or on Saturday. You never really know.
+I'm not sure I want to talk...
```

Comment:
Honestly, 90% of the time I invite people to check out Barcamp, their not wanting to talk (either they don't think they have anything worth talking about, or they're uncomfortable in a public speaking role, or they're afraid nobody will show up) is the first question they have after I explain the baseline of how it works. So I find headlining that's it's OK not to talk is rather important.